From c948904967fb49e0f6195b8b748bbfd346d4721f Mon Sep 17 00:00:00 2001
From: Xinhuang Li <buluess.li@rock-chips.com>
Date: Tue, 8 May 2018 10:43:42 +0800
Subject: [PATCH] video: rockchip: iep: fix compile warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

fix the warning:
drivers/video/rockchip/iep/hw_iep_reg.c:1284:6: warning: no
previous prototype for ‘iep_config_misc’ [-Wmissing-prototypes]

Change-Id: I56d0b775cdb62e4566d81b288fe373299b9bbc4c
Signed-off-by: Xinhuang Li <buluess.li@rock-chips.com>
---
 drivers/video/rockchip/iep/hw_iep_reg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/iep/hw_iep_reg.c b/drivers/video/rockchip/iep/hw_iep_reg.c
index 0683ae644a2b..331a50865045 100644
--- a/drivers/video/rockchip/iep/hw_iep_reg.c
+++ b/drivers/video/rockchip/iep/hw_iep_reg.c
@@ -1281,7 +1281,7 @@ void iep_config_frame_end_int_en(void *base)
 	IEP_REGB_FRAME_END_INT_EN(base, 1);
 }
 
-void iep_config_misc(struct IEP_MSG *iep_msg)
+static void iep_config_misc(struct IEP_MSG *iep_msg)
 {
 //	IEP_REGB_V_REVERSE_DISP();
 //	IEP_REGB_H_REVERSE_DISP();
-- 
2.35.3

